channeldb/payments.go

```diff
@@ -35,24 +35,21 @@ var (
 	//      |        |
 	//      |        |--payment-htlcs-bucket (shard-bucket)
 	//      |        |        |
-	//      |        |        |-- <htlc attempt ID>
-	//      |        |        |       |--htlc-attempt-info-key: <htlc attempt info>
-	//      |        |        |       |--htlc-settle-info-key: <(optional) settle info>
-	//      |        |        |       |--htlc-fail-info-key: <(optional) fail info>
+	//      |        |        |-- ai<htlc attempt ID>: <htlc attempt info>
+	//      |        |        |-- si<htlc attempt ID>: <(optional) settle info>
+	//      |        |        |-- fi<htlc attempt ID>: <(optional) fail info>
```

Comment:
Because this optimization isn't the lightest with a full payments db migration involved, I think it would be good to add to the PR description in which scenarios the speed up will be felt and how much it is roughly.